docs/HowToDevelopThingsUsingSTDK.md

```diff
@@ -0,0 +1,256 @@
+# How to develop things using STDK based TizenRT   
+
+STDK is one of Develop Kit(STDK) for Smart Things Eco-system.   
+This guide shows an example how to enable STDK and develop things.   
+
+So now, it will explain how to develop Led Things of SmartThings and Control it through SmartThings Mobile app.   
+
+## Prerequisites   
+The goal of this guide is to execute a sample application on esp32 board. so this doc will describe an experience how to build and run a STDK App. For this example, It need a ESP-WROVER-KIT board.   
```

Comment:
1. "so this doc .." -- Start sentence with a  capital latter
2. Replace the sentence with "This section will describe the pre-requisite steps to install and build STDK for the ESP-WROVER-KIT board".

Reply:
Has Changed.